pom.xml

```diff
@@ -1972,6 +1972,26 @@
 
     <!-- Profiles -->
     <profiles>
+        <profile>
+            <id>quick-build</id>
+            <activation>
+                <property>
+                    <name>quickly</name>
```

Comment:
Instead of !skip.ee8 we could also use !quickly here:

https://github.com/wildfly/wildfly/blob/main/pom.xml#L1980

Building those modules is really pretty pointless now.

Perhaps here as well:

https://github.com/wildfly/wildfly/blob/main/ee-9/pom.xml#L127

Whether building WFP is appropriate for 'quick-build' is debatable. How appropriate it is varies over time, but my instinct is it's not really necessary to build it.

Reply:
The WFP decision should be informed by data; i.e. how much time it adds.

Reply:
Re both of the above thoughts, doing that would prevent using those profile on the CI servers.

It's a shame profile activation doesn't support and/or configuration.

Reply:
My machine feels better rested on Monday. I'm getting even better times :).

With both above changes 01:13 (199 modules). WFP included: 01:30 (207 modules), current state 01:29 (236 modules).

Most of the modules are compiled pretty quickly. Most time is spent in the galleon provisioning at the end. WFP is provisioned together with ee-dist and dist at the same time (different thread), so there is not so much difference.

Reply:
@bstansberry so I suppose we can go without WFP for now and if required we will add it back?

Reply:
@xstefank +1.

TBH I don't know if we are actually using -Dskip.ee8 on CI jobs. Maybe I added the profile but never got around to changing CI to use it. If I had set CI to use it, I think what I asked Yeray to fix for bonus points in https://github.com/wildfly/wildfly/pull/16173#pullrequestreview-1142880733 would have resulted in obvious CI failures.

If we're not using it on CI, breaking it is fine. :)